app/assets/javascripts/application.js

```diff
@@ -33,6 +33,6 @@
 //= require configure_quill_editor
 //= require plugins_toggle
 
-//= require 'page'
+//= require('page');
```

Comment:
wait what?? this fixed the error? It should be `require('page')` not `//= require('page')`. The issue was the sprockets include. I have no idea what this does now or why it works.